napari/layers/image/_image_constants.py

```diff
@@ -63,3 +65,4 @@ class Rendering(StringEnum):
     MIP = auto()
     MINIP = auto()
     ATTENUATED_MIP = auto()
+    AVG = auto()
```

Comment:
I wonder if this should be `AVERAGE` instead of `AVG`? Curious what @jni thinks?

Reply:
I'm 100% Team AVERAGE. :joy:

Reply:
done and done! :+1: